test/AWS.Deploy.CLI.UnitTests/ApplyPreviousSettingsTests.cs

```diff
@@ -67,8 +77,8 @@ public async Task ApplyApplicationIAMRolePreviousSettings(bool createNew, string
         }
 
         [Theory]
-        [InlineData(true, false, null)]
-        [InlineData(false, true, null)]
+        [InlineData(true, false, "")]
+        [InlineData(false, true, "")]
```

Comment:
You are changing the intent of the test here. We should mark `vpcId` nullable here.

Reply:
For the purpose of this test, the Assertion that vpcId will be null will always be false since GetOptionSettingValue now always returns a value. It will never be null. If you think the new test is not meaningful, we can remove it.